cypress/integration/BasicSearchPage/BasicSearchPage.feature

```diff
@@ -0,0 +1,219 @@
+    When user types "22182" in "U.S. Zip Code" field
+    And user clicks on "Find Trials" button
+    Then the search is executed and results page is displayed
+		And trial info displays "Results 1-10  of 1130 for your search "
+    And the criteria table displays the following
+		| Category                      | Selection                 |
+    | Near ZIP Code                 | 22182             |
+		And the url query has the following corresponding code
+      | parameter | value |
+      | z         | 22182 |
+      | loc       | 1     |
+      | rl        | 1     |
+
+      Scenario: Negative: User is not able to search for an incorrect zip code
+    Given the user navigates to "/about-cancer/treatment/clinical-trials/search"
+    Then the page title is "Find NCI-Supported Clinical Trials"
+    And "U.S. Zip Code" form section is displayed
+    When user types "999g9" in "U.S. Zip Code" field
+    Then alert "Please enter a valid 5 digit U.S. zip code" is displayed in "U.S. Zip Code" section
+    And user clicks on "Find Trials" button
+    Then the search is not executed and path is "/about-cancer/treatment/clinical-trials/search"
+
+
+Scenario: User is able to search for a trials using age and keyword
+Given screen breakpoint is set to "desktop"
+    Given the user navigates to "/about-cancer/treatment/clinical-trials/search"
+    Then the page title is "Find NCI-Supported Clinical Trials"
+    When user types "breast cancer" in "Cancer Type/Keyword" field
+    And user selects "Breast Cancer" from dropdown
+    And user types "40" in "Age" field
+    And user clicks on "Find Trials" button
+    Then the search is executed and results page is displayed
+		And the criteria table displays the following
+		| Category                      | Selection                 |
+    | Primary Cancer Type/Condition | Breast Cancer             |
+		| Age                           | 40                        |
+    And results info has text "Results 1-10  of 894 for your search "
+    And the url query has the following corresponding code
+      | parameter | value |
+      | loc       | 0     |
+      | rl        | 1     |
+      | t         | C4872 |
+      | a         | 40    |
+
+
+Scenario: User is able to search for all trials
+Given the user navigates to "/about-cancer/treatment/clinical-trials/search"
+    Then the page title is "Find NCI-Supported Clinical Trials"
+    And user clicks on "Find Trials" button
+    Then the search is executed and results page is displayed
+   And trial info displays "Results 1-10  of 6821 for your search for: \"all trials\"   |  Start Over"
+    And the url query has the following corresponding code
+      | parameter | value |
+      | loc       | 0     |
+      | rl        | 1     |
+
+Scenario: User has an option to go to search results url directly and modify search
+Given the user navigates to "/about-cancer/treatment/clinical-trials/search/r?loc=0&rl=1&t=C4872"
```

Comment:
make the url` /about-cancer/treatment/clinical-trials/search/r?a=30&loc=1&q=aids&rl=1&z=22182`